### DISCUSSION TITLE
Add blocs in /condense

### DISCUSSION THREAD

**Discussion by Darkseno** [Zrips/CMI#7468] [Feature Requests]
Hello,

On my server, some ranks have access to the /condense command except that some blocks like the nether wart or the melon can't be /condense I find that it's not logical because in vanilla minecraft, the construction of these blocks exists with 9 nether wart or 9 melon slide.

It annoys a lot my players who make big farms, would it be possible to add these blocks or just add an option to add /condense again?

**Comment by GcGoat** [comment]
Condense only works if you have recipes for condense and uncondense actions, so if you want to have those blocks then you will need to make custom recipe to convert 9 melon slices into a melon, same with nether wart or any other block you might want to include

**Reply by Darkseno** [reply]
I've done it before and it doesn't work, every time it tells me that there is no object to condense

**Reply by Darkseno** [reply]
![melon](https://user-images.githubusercontent.com/99391020/209675296-57046535-1c3f-4668-a589-f2dc584b2f94.jpg)

**Comment by Zrips** [comment]
You need to make shapeless recipe like this one
![image](https://user-images.githubusercontent.com/10416201/209806622-7b59c0d9-e19d-4b29-86e8-522be7027d91.png)
converting melon into 9 slices, as there is one to convert slices into melon.

**Reply by Darkseno** [reply]
Thanksssssssssssss
